filesystems/general/pjd-fstest/runtest.sh

```diff
@@ -0,0 +1,254 @@
+#!/bin/bash
+# vim: dict=/usr/share/rhts-library/dictionary.vim cpt=.,w,b,u,t,i,k
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   runtest.sh of /kernel/filesystems/general/pjd-fstest
+#   Description: Tests POSIX features of file system
+#   Author: Zorro Lang <zlang@redhat.com>
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+#   Copyright (c) 2019 Red Hat, Inc. All rights reserved.
+#
+#   This copyrighted material is made available to anyone wishing
+#   to use, modify, copy, or redistribute it subject to the terms
+#   and conditions of the GNU General Public License version 2.
+#
+#   This program is distributed in the hope that it will be
+#   useful, but WITHOUT ANY WARRANTY; without even the implied
+#   warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#   PURPOSE. See the GNU General Public License for more details.
+#
+#   You should have received a copy of the GNU General Public
+#   License along with this program; if not, write to the Free
+#   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+#   Boston, MA 02110-1301, USA.
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+# Include rhts environment and beaker library
+. /usr/bin/rhts-environment.sh || exit 1
+. /usr/share/rhts-library/rhtslib.sh || exit 1
+
+
+# Test command, not the pipe (tee ...)
+set -o pipefail
+
+#prepare global values
+test -z "$TEST_PARAM_DEVICE" || TEST_DEV="${TEST_PARAM_DEVICE}"
+test -z "$TEST_PARAM_TESTMNT" || TEST_MNT="${TEST_PARAM_TESTMNT}"
+test -z "$TEST_PARAM_FSTYPE" && TEST_PARAM_FSTYPE="$TEST_PARAM_FSTYP" # Backwards compatiblity
+if [ -n "$TEST_PARAM_FSOPTS" ];then
+	FSOPTS="-o $TEST_PARAM_FSOPTS"
+fi
+if [ -n "$TEST_PARAM_FSTYPE" ]; then
+	FSTYPES="$TEST_PARAM_FSTYPE"
+else
+	FSTYPES="ext3 ext4 xfs"
+fi
+FSTYPE=""
+TestsDir=""
+TmpDir=""
+
+# make sure the arguments all lower
+FSTYPE=$(echo ${FSTYPE}|tr [:upper:] [:lower:])
+FSOPTS=$(echo ${FSOPTS}|tr [:upper:] [:lower:])
+
+RHEL_DISTRO=$(rlGetDistroRelease)
+RHEL_DISTRO=${RHEL_DISTRO:0:1}
+LOCAL_ARCH=`uname -m`
+
+
+# select tool to manage package, which could be "yum" or "dnf"
+select_yum_tool() {
+    if [ -x /usr/bin/dnf ]; then
+        echo "/usr/bin/dnf"
+    elif [ -x /usr/bin/yum ]; then
+        echo "/usr/bin/yum"
+    else
+        return 1
+    fi
+
+    return 0
+}
+
+
+yum=$(select_yum_tool)
+
+check_supported_fs() {
+#	case ${FSTYPE} in
```

Comment:
Maybe delete this commented-out line?

Reply:
correct, some left-over  comments  should be deleted.